src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/metrics/ExceptionsAndErrors.java

```diff
@@ -37,7 +37,12 @@
   /**
    * When persisting an RCA throws an exception.
    */
-  EXCEPTION_IN_PERSIST("ExceptionInPersist", "namedCount", Statistics.NAMED_COUNTERS);
+  EXCEPTION_IN_PERSIST("ExceptionInPersist", "namedCount", Statistics.NAMED_COUNTERS),
+
+  /**
+   * When persisting an action, the persistable throws an exception when it is unable to write to DB.
+   */
+  EXCEPTION_IN_WRITE_TO_SQLITE("ExceptionInWriteToSqlite", "namedCount", Statistics.NAMED_COUNTERS);
```

Comment:
Can we call out the difference between `EXCEPTION_IN_WRITE_TO_SQLITE` and `EXCEPTION_IN_PERSIST`and why we require both ? 
The thing with named counters is you can bucketize over the same enum. So, say `EXCEPTION_IN_PERSIST` metric is reported when `EX1` is thrown and also when `EX2` is thrown. You can pass that along and in the reported metrics you will see both buckets with counts.

Reply:
I think we can use persist for both. As it is named counter, we will have a counter for each one.